app/controllers/training_controller.rb

```diff
@@ -68,8 +69,26 @@ def find_slide
     redirect_to "/training/#{training_library.slug}/#{training_module.slug}/#{training_slide.slug}"
   end
 
+  def fetch_training_mode
+    render json: { training_mode: current_training_mode }, status: :ok
```

Comment:
Per our discussion today, I think the training mode can be included in the training layout haml file as an attribute of the react root, so that making a separate call just for this piece of data is not necessary.

Reply:
sure, i have done it